pom.xml

```diff
@@ -119,10 +120,105 @@
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>27.0.1-jre</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>connect-runtime</artifactId>
+            <version>${kafka.version}</version>
+            <classifier>test</classifier>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>connect-runtime</artifactId>
+            <version>${kafka.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <version>${kafka.version}</version>
+            <classifier>test</classifier>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_2.11</artifactId>
+            <version>${kafka.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_2.11</artifactId>
+            <type>test-jar</type>
+            <classifier>test</classifier>
+            <scope>test</scope>
+            <version>${kafka.version}</version>
+        </dependency>
     </dependencies>
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>${version.docker.maven.plugin}</version>
+                <configuration>
+                    <watchInterval>500</watchInterval>
+                    <logDate>default</logDate>
+                    <verbose>true</verbose>
+                    <images>
+                        <image>
+                            <alias>elastic-integration-test</alias>
+                            <!--"name" is in local images if "build" is defined, else will look
+                            for an image in a remote repo-->
+                            <name>confluentinc/elastic-integration</name>
+                            <build>
+                                <dockerFile>${project.basedir}/src/test/resources/Dockerfile</dockerFile>
+                            </build>
+                            <run>
+                                <hostname>elastichost</hostname>
+                                <ports>
+                                    <port>9200:9200</port>
+                                </ports>
+                                <wait>
+                                    <tcp>
+                                        <host>localhost</host>
+                                        <ports>
+                                            <port>9200</port>
```

Comment:
with the hardcoded port, this test might fail if that port is not available. would it be possible to look up the port number in the test? that way, the container can pick up any available port, and the test can discover it at runtime. this also opens up opportunities for running parallel tests.

Reply:
This is a good point, but I cannot figure out how to do it, and in Debezium project they also use hardcoded values. I'm not sure it's worth the effort to figure this out